omod/src/main/java/org/openmrs/module/adminui/page/controller/myaccount/ChangeDefaultsPageController.java

```diff
@@ -36,7 +36,8 @@ public void get(PageModel pageModel) {
         userDefaults.setDefaultLocale(props.get(OpenmrsConstants.USER_PROPERTY_DEFAULT_LOCALE));
         userDefaults.setProficientLocales(props.get(OpenmrsConstants.USER_PROPERTY_PROFICIENT_LOCALES));
         pageModel.addAttribute("userDefaults", userDefaults);
-        pageModel.addAttribute("locales", Context.getAdministrationService().getPresentationLocales());
+        pageModel.addAttribute("primaryLocale", Context.getAdministrationService().getPresentationLocales());
+        pageModel.addAttribute("proficientLocale", Context.getAdministrationService().getAllowedLocales());
```

Comment:
I was initially confused that these are singular and not plural. Could you do a quick followup fix and rename these to either "presentationLocales" and "allowedLocales", or else call them "primaryLocaleOptions" and "proficientLocaleOptions"?